explorer/cmd/cmd.go

```diff
@@ -13,9 +13,8 @@ func GetCommands() []*plugin.Command {
 			Aliases: []string{"e"},
 			Commands: []*plugin.Command{
 				{
-					Use:     "gex",
-					Short:   "Run gex explorer",
-					Aliases: []string{"g"},
```

Comment:
I  think we need the alias here so we can show it in the usage

Reply:
that's true, reverting!